test/Cpp/src/multiport/Broadcast.lf

```diff
@@ -0,0 +1,27 @@
+target Cpp;
+
+reactor Source {
```

Comment:
I think at least the `Source` reactor (but also a parametrized version of `Sink`) can be imported for a bit of a more minimalistic design. This is by no means a critical issue though.

Reply:
Yep, having better guidelines for writing tests and establishing a set of basic library reactors would be nice.